Source/WebCore/rendering/BackgroundPainter.cpp

```diff
@@ -453,7 +454,7 @@ void BackgroundPainter::paintFillLayer(const Color& color, const FillLayer& bgLa
         // Multiline inline boxes paint like the image was one long strip spanning lines. The backgroundImageStrip is this fictional rectangle.
         auto imageRect = backgroundImageStrip.isEmpty() ? scrolledPaintRect : backgroundImageStrip;
         auto paintOffset = backgroundImageStrip.isEmpty() ? rect.location() : backgroundImageStrip.location();
-        auto geometry = calculateBackgroundImageGeometry(m_renderer, m_paintInfo.paintContainer, bgLayer, paintOffset, imageRect);
+        auto geometry = calculateBackgroundImageGeometry(m_renderer, m_paintInfo.paintContainer, bgLayer, paintOffset, imageRect, m_overrideClip);
```

Comment:
There's a bug here. `m_overrideClip` is used to override `background-clip`, but here you're using to also override `background-origin`. We need to keep these separate.

Reply:
I am fixing via https://github.com/WebKit/WebKit/pull/34144

Reply:
Thanks for this fix! In the context of tables it didn't make a difference, but I could've predicted that future commits like 21c075d2c09ac20f39ea272e51173655478284c1 might want only part of the functionality.